.github/ISSUE_TEMPLATE/retrospective.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Retrospective
+about: Template to submit a retrospective
+title: "[SUBMISSION] Title of the paper"
+labels: 'retrospective'
+assignees: ''
```

Comment:
I would suggest adding the keys `paper_name` and `paper_url` in the template itself, so that we can render them in Liquid tags for the Published Retrospectives Page.

Reply:
The key `tags` can also be added which would be a comma separated tag, which will be used later for search

Reply:
This is a bit confusing, but this section is not keys for Jekyll, they are keys for github's prepolutation of issue's fields. In any case, we should probably not use the issues for retrospectives as discussed in the email thread. I suggest we find some consensus on the email thread and then close this PR is we decided not to go with issues to submit retrospectives. In this case we could also drop the bug and feature templates.